Neos.ContentRepository/Classes/Command/NodeCommandControllerPlugin.php

```diff
@@ -291,25 +295,32 @@ protected function askBeforeExecutingTask($question, \Closure $task)
      *
      * @param string $workspaceName Name of the workspace to consider
      * @param boolean $dryRun Simulate?
+     * @param boolean $summary Only Status?
      * @param NodeType $nodeType Only for this node type, if specified
-     * @return void
+     * @return integer
      */
-    protected function createMissingChildNodes($workspaceName, $dryRun, NodeType $nodeType = null)
+    protected function createMissingChildNodes($workspaceName, $dryRun, $summary, NodeType $nodeType = null)
     {
+        $errorCount = 0;
+
         if ($nodeType !== null) {
             $this->output->outputLine('Checking nodes of type "%s" for missing child nodes ...', [$nodeType->getName()]);
-            $this->createChildNodesByNodeType($nodeType, $workspaceName, $dryRun);
+            $errorCount = $this->createChildNodesByNodeType($nodeType, $workspaceName, $dryRun, $summary);
         } else {
             $this->output->outputLine('Checking for missing child nodes ...');
             foreach ($this->nodeTypeManager->getNodeTypes() as $nodeType) {
                 /** @var NodeType $nodeType */
                 if ($nodeType->isAbstract()) {
                     continue;
                 }
-                $this->createChildNodesByNodeType($nodeType, $workspaceName, $dryRun);
+                $errorCount += $this->createChildNodesByNodeType($nodeType, $workspaceName, $dryRun, $summary);
             }
         }
 
+        if ($summary) {
+            return $errorCount;
+        }
```

Comment:
The `@return` annotation is `integer` but this only returns a value if `summary` is set?